detekt_custom.yml

```diff
@@ -1245,6 +1245,7 @@ datadog:
       - "kotlin.text.StringBuilder()"
       - "kotlin.text.buildString(kotlin.Function1)"
       - "kotlin.text.buildString(kotlin.Int, kotlin.Function1)"
+      - "java.lang.StringBuilder.clear()"
```

Comment:
this has to be moved to the right position in this file: it doesn't belong to the `Kotlin String` region and once proper region found, the position should be in the alphabetical order.

Reply:
I thought that if it's extension function it should be in kotlin space, anyway, moved

Reply:
Then it should be `kotlin.text.StringBuilder.clear()` here.

Reply:
`java.lang.StringBuilder.clear()` is how detekt sees it. Probably there is some issue in `UnsafeThirdPartyFunctionCall` but adding `kotlin.text.StringBuilder.clear()` to whitelist will not help.